graphql_compiler/compiler/expressions.py

```diff
@@ -885,6 +949,47 @@ def to_gremlin(self):
                                   left=self.left.to_gremlin(),
                                   right=self.right.to_gremlin())
 
+    def to_cypher(self):
+        """Return a unicode object with the Cypher representation of this expression."""
+        self.validate()
+
+        # The Cypher versions of some operators require an inverted order of arguments.
+        regular_operator_format = u'({left} {operator} {right})'
+        inverted_operator_format = u'({right} {operator} {left})'
+        negated_inverted_operator_format = u'(NOT ({right} {operator} {left}))'
+        intersects_operator_format = u'any(_ {operator} {left} WHERE _ {operator} {right})'
```

Comment:
Nit: capitalize ANY (I've tested that this is case-insensitive and the other keywords are all capitalized).

Reply:
Looks good to me once this is changed!

Reply:
Ah, good point-- I'm too used to [SQL formatting](https://www.w3schools.com/sql/sql_ref_sqlserver.asp). In that case, everything looks good to me!

Reply:
Thanks for looking it over. If you'd like to express your acceptance by formally submitting an accepting review, I'll be able to merge this PR :)

Reply:
done!